src/python/WMCore/ReqMgr/CherryPyThreads/StatusChangeTasks.py

```diff
@@ -64,6 +64,39 @@ def moveToArchivedForNoJobs(reqmgrSvc, wfStatusDict, logger):
                 for nextStatus in nextStatusList:
                     reqmgrSvc.updateRequestStatus(wf, nextStatus)
                     count += 1
+        logger.info("Total aborted-completed: %d", count)
+
+    return
+
+
+def moveToArchived(wmstatsSvc, reqmgrSvc, archiveDelayHours, logger):
+    """
+    Handle normal-archived transition.
+    By checking AgentJobInfo status we can check whether all the agent deleted the data.
+    TODO: still need to handle the case whether agent couch is not updated for a while and agent data gets deleted
+    """
+
+    currentTime = int(time.time())
+    threshold = archiveDelayHours * 3600
+    count = 0
+
+    statusTransition = {"announced": "normal-archived",
+                        "aborted-completed": "aborted-archived",
+                        "rejected": "rejected-archived"}
+
+    inputConditon = {"RequestStatus": ["announced"], "AgentJobInfo": "NO_KEY"}
```

Comment:
My point was that you also set `inputConditon` in L91, so this one simply gets overwritten and never used.